### PR TITLE
"iframe with identical name serve equivalent purpose" (4b1c6c): clean up examples descriptions

### PR DESCRIPTION
**Pull request by Jym77**
Cleaning up and polishing the examples descriptions.

Closes issue(s):

- preparatory work before starting #769 

Need for Final Call:
This will not require a Final Call (changes to example descriptions)

---

### **When creating PR:**

- Make sure you requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**

- Add yourself (and co-authors) as "Assignees" for PR.
- Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)
- Optionally request feedback from anyone in particular by assigning them as "Reviewers".

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/iframe-identical-name-equivalent-purpose-4b1c6c.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ Multiple `iframe` elements in the document having different `aria-labelledby` de
 
 #### Inapplicable Example 5
 
-`iframe` having the same `title` within a given document tree, but one of them is not [included in the accessibility tree][].
+Both `iframe` elements have the same [accessible name][] (given by the `title` attribute) within a the same [document tree][], but one of them is not [included in the accessibility tree][]. Therefore, there is no set of two or more `iframe` elements with the same [accessible name][] and [included in the accessibility tree][] and the rule is inapplicable.
```
Comment (carlosapaduarte) [inline _rules/iframe-identical-name-equivalent-purpose-4b1c6c.md]: within a the same > within the same

Comment (carlosapaduarte) [inline _rules/iframe-identical-name-equivalent-purpose-4b1c6c.md]: same [accessible name][] and [included in the accessibility tree][] > same [accessible name][] [included in the accessibility tree][]

File: _rules/iframe-identical-name-equivalent-purpose-4b1c6c.md [hunk 2]
```diff
@@ -241,15 +241,17 @@ Multiple `iframe` elements in the document having different `aria-labelledby` de
 
 #### Inapplicable Example 7
 
-`alt` cannot be used to provide [accessible name][] for iframe.
+The `alt` attribute does not provide an [accessible name][] for `iframe` elements. Therefore, these `iframe` elements do not have an [accessible name][] and the rule is inApplicable.
```
Comment (carlosapaduarte) [inline _rules/iframe-identical-name-equivalent-purpose-4b1c6c.md]: inApplicable > inapplicable

File: _rules/iframe-identical-name-equivalent-purpose-4b1c6c.md [hunk 3]
```diff
@@ -259,7 +261,7 @@ Does not apply to `object` elements.
 
 #### Inapplicable Example 9
 
-No [accessible name][] is provided
+The `iframe` element do not have [accessible names][accessible name].
```
Comment (carlosapaduarte) [inline _rules/iframe-identical-name-equivalent-purpose-4b1c6c.md]: element > elements

File: _rules/iframe-identical-name-equivalent-purpose-4b1c6c.md (hunk 1 above)
Comment (Jym77) [reply _rules/iframe-identical-name-equivalent-purpose-4b1c6c.md]: It's the `iframe`, not the accessible name, that need to be included in the accessibility tree.
I tried to improve the formulation.

**Pull request merged**